src/lib/cssShorthandToClasses.ts

```diff
@@ -33,10 +33,7 @@ function generateBaseClasses(
     const sides = value.split(' ');
 
     sides.forEach((v, index) => {
-      // Exception for border width since we want to default borders to zero.
-      if (v !== '0' || attribute === 'border-width') {
-        classes.push(`${attribute}-${shorthand[sides.length][index]}-${v}`);
-      }
+      classes.push(`${attribute}-${shorthand[sides.length][index]}-${v}`);
```

Comment:
Since this exception is specifically for border-width, I'm unclear how this affects spacing classes. is there something I'm missing?

Reply:
nevermind, I misread this and now it makes perfect sense. @nathanyoung